src/main/java/frc/robot/mechanisms/pivot/Pivot.java

```diff
@@ -11,16 +11,18 @@ public class Pivot extends Mechanism {
     public class PivotConfig extends Config {
 
         /* Pivot constants in motor rotations */
-        public final double maxRotation = 23;
+        public final double maxRotation = -23; // starting at max pos
         // happen
-        public final double minRotation = 0;
+        public final double minRotation = 0; // max motor rotations down
 
-        /* Pivot positions in percentage of max rotation || 0 is vertical? */
-        public final int score = 80;
-        public final int halfScore = 50;
-        public final int test = 65;
+        /* Pivot positions in angle measure(degrees) || 0 is min/flat */
```

Comment:
while we're at it we can go ahead and change these to be doubles, we'll most likely end up needing decimal setpoints